internal/commands/pull_buildpack.go

```diff
@@ -0,0 +1,45 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/internal/style"
+	"github.com/buildpacks/pack/logging"
+)
+
+type PullBuildpackFlags struct {
+	BuildpackRegistry string
+}
+
+func PullBuildpack(logger logging.Logger, cfg config.Config, client PackClient) *cobra.Command {
+	var opts pack.PullBuildpackOptions
+	var flags PullBuildpackFlags
+
+	cmd := &cobra.Command{
+		Use:     "pull-buildpack <uri>",
+		Args:    cobra.ExactArgs(1),
+		Short:   prependExperimental("Pull the buildpack and store it locally"),
+		Example: "pack pull-buildpack example/my-buildpack@1.0.0",
```

Comment:
nit: Could you add a `Long:` description clarifying why people would use it, and what valid inputs look like?